### PR TITLE
Added Vaadin Demos

### PR DESCRIPTION
**Pull request by svenruppert**

### CODE REVIEW

File: examples/vaadin-lts-v08-simple/pom.xml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
```
Comment (brunobat) [inline examples/vaadin-lts-v08-simple/pom.xml]: Please remove the parent dependency in here and the other poms. We try to keep the examples self contained to make the required dependencies explicit. That can save a lot of time to whoever is using the example.

File: examples/vaadin-lts-v08-simple/pom.xml [hunk 2]
```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (brunobat) [inline examples/vaadin-lts-v08-simple/pom.xml]: Please add the Apache license header to the pom.xml files too.

File: examples/vaadin-lts-v10-simple/pom.xml [hunk 3]
```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (brunobat) [inline examples/vaadin-lts-v10-simple/pom.xml]: I've tried to compile the vaadin-lts-v10-simple/ and vaadin-vxx-simple/ examples and I get this error on both:

> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-war-plugin:3.1.0:war (default-war) on project vaadin-lts-v10-simple: Error assembling WAR: webxml attribute is required (or pre-existing WEB-INF/web.xml if executing in update mode) -> [Help 1]

**Comment by svenruppert** [comment]
@brunobat ok, changes are pushed

File: examples/vaadin-lts-v08-simple/pom.xml (hunk 2 above)
Comment (brunobat) [reply examples/vaadin-lts-v08-simple/pom.xml]: Thanks

**Comment by svenruppert** [comment]
Hi @brunobat , anything I can do to get it merged ?

**Comment by radcortez** [comment]
It is merged :)

**Comment by svenruppert** [comment]
@radcortez thx :-)

**Pull request merged**